Reconstruction/RecCalorimeter/src/components/CreateCaloCells.h

```diff
@@ -56,25 +58,28 @@ class CreateCaloCells : public GaudiAlgorithm {
   ToolHandle<ICalorimeterTool> m_geoTool;
 
   /// Calibrate to EM scale?
-  bool m_doCellCalibration;
+  Gaudi::Property<bool> m_doCellCalibration{this, "doCellCalibration", true, "Calibrate to EM scale?"};
   /// Add noise to cells?
-  bool m_addCellNoise;
+  Gaudi::Property<bool> m_addCellNoise{this, "addCellNoise", true, "Add noise to cells?"};
   /// Save only cells with energy above threshold?
-  bool m_filterCellNoise;
+  Gaudi::Property<bool> m_filterCellNoise{this, "filterCellNoise", false, "Save only cells with energy above threshold?"};
   /// Handle for calo hits (input collection)
   DataHandle<fcc::CaloHitCollection> m_hits;
   /// Handle for calo cells (output collection)
   DataHandle<fcc::CaloHitCollection> m_cells;
   /// Name of the detector readout
-  std::string m_readoutName;
-  /// Name of active volumes
-  std::string m_activeVolumeName;
+  Gaudi::Property<std::string> m_readoutName{this, "readoutName", "ECalHitsPhiEta", "Name of the detector readout"};
+  /// Name of active volumes (material name)
```

Comment:
Name of active volume == material name for the current implementation, but that doesn't always need to be true.